docs/InstallGuide.md

```diff
@@ -92,18 +81,59 @@ Modify the following section in /etc/openvnet/webapi.conf on the webapi node.
       addr {
         protocol "tcp"
         host "192.168.100.2"
+        public ""
         port 9101
       }
     }
```

Comment:
Same comments as vnmgr and vna above. Use loopback address and \`\` around the file name.